bot/exts/core/ping.py

```diff
@@ -20,7 +21,7 @@ async def ping(self, ctx: commands.Context) -> None:
         embed = Embed(
             title=":ping_pong: Pong!",
             colour=Colours.bright_green,
-            description=f"Gateway Latency: {round(self.bot.latency * 1000)}ms",
+            description=f"Gateway Latency: `{round(self.bot.latency * 1000)}`ms",
```

Comment:
![image](https://github.com/python-discord/sir-lancebot/assets/22353562/ad185e38-0152-494c-84cb-d8d7f08804f6)

I think both ways look fine, so I don't really think it's worth changing unless there's a good reason.

Reply:
Fair enough. I guess it's personal  preference but definitely not a necessary change.